website/docs/components/table/partials/guidelines/guidelines.md

```diff
@@ -155,4 +155,49 @@ Row placement determines the visual styling based on where the row is placed rel
 - We recommend using `medium` cell density by default.
 - If content is complex or a smaller data set (e.g., a Table of basic user data), `tall` cell density allows for more breathing room surrounding the content.
 - If content is largely string/text-based, `short` allows for more content to be displayed within the page.
-- While denser content allows for more rows to be displayed within a single page, it also makes comprehension and scanning more difficult.
\ No newline at end of file
+- While denser content allows for more rows to be displayed within a single page, it also makes comprehension and scanning more difficult.
+
+## Multi-Selection
+
+A multi-select table includes checkboxes enabling users to select multiple rows in a table for purposes of performing bulk operations. Checking or unchecking the checkbox in the table header either selects or deselects the checkboxes on each row in the table body. Individual checkboxes in the rows can also be selected or deselected.
+
+Multi-selection consist of: 
+
+1. `Header Column / Select`: Used in the table's header row. This acts as the parent checkbox, allowing the selection or deselection of all child rows on a single page simultaneously.
```

Comment:
@andgen404 can confirm, but I believe "page" is being used because if the table is paginated, then the checkbox in the header will only select the visible rows (so a single "page"). That's at least how I've understood this to work as it's been explained in reviews, but if that's not the intended interaction, then "table" probably makes more sense.

Reply:
Yeah +1 to what @heatherlarsen said, I used a page here for multiple pages in a table. The Header Column / Select would select all rows in that page view.